SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -135,6 +135,6 @@ void USpatialSender::SendAuthorityIntentUpdate(const AActor& Actor, VirtualWorke
 
 	if (NetDriver->SpatialDebuggerSystem.IsValid())
 	{
-		NetDriver->SpatialDebuggerSystem->ActorAuthorityIntentChanged(EntityId, NewAuthoritativeVirtualWorkerId);
+		NetDriver->SpatialDebuggerSystem->ActorAuthorityIntentChanged(EntityId, NewAuthoritativeVirtualWorkerId, *GetNameSafe(&Actor));
```

Comment:
Couldn't we validate the virtual worker id at the top of this function instead? Seems safer/better than going through with a auth update that will fail.